app/views/stats/index.phtml

```diff
@@ -161,37 +160,37 @@ $entryValues = array_values($entryData);
 ?>
 
 <div class="stat half">
-	<h2><?= _t('admin.stats.feed_per_category') ?></h2>
-	<div>
-		<canvas id="statsFeedsPerCategory"></canvas>
-		<script class="jsonData-stats" type="application/json">
-		<?= json_encode([
-			'canvasID' => 'statsFeedsPerCategory',
-			'charttype' => 'doughnut',
-			'data' => $feedValues,
-			'labels' => $feedLabels,
-			'backgroundColor' => $feedColors,
-		], JSON_UNESCAPED_UNICODE); ?>
-		</script>
-	</div>
+    <h2><?= _t('admin.stats.feed_per_category') ?></h2>
+    <div>
+        <canvas id="statsFeedsPerCategory"></canvas>
+        <script class="jsonData-stats" type="application/json">
+        <?= json_encode([
+            'canvasID' => 'statsFeedsPerCategory',
+            'charttype' => 'doughnut',
+            'data' => $feedValues,
+            'labels' => $feedLabels,
+            'backgroundColor' => $feedColors,
+        ], JSON_UNESCAPED_UNICODE); ?>
+        </script>
+    </div>
 </div>
 
 <div class="stat half">
-	<h2><?= _t('admin.stats.entry_per_category') ?></h2>
-	<div>
-		<canvas id="statsEntriesPerCategory"></canvas>
-		<script class="jsonData-stats" type="application/json">
-			<?= json_encode([
-				'canvasID' => 'statsEntriesPerCategory',
-				'charttype' => 'doughnut',
-				'data' => $entryValues,
-				'labels' => $entryLabels,
-				'backgroundColor' => $entryColors,
-				], JSON_UNESCAPED_UNICODE); ?>
-				</script>
-			</div>
-		</div>  
-	</div>
+    <h2><?= _t('admin.stats.entry_per_category') ?></h2>
+    <div>
+        <canvas id="statsEntriesPerCategory"></canvas>
+        <script class="jsonData-stats" type="application/json">
+            <?= json_encode([
+                'canvasID' => 'statsEntriesPerCategory',
+                'charttype' => 'doughnut',
+                'data' => $entryValues,
+                'labels' => $entryLabels,
+                'backgroundColor' => $entryColors,
+                ], JSON_UNESCAPED_UNICODE); ?>
+                </script>
+            </div>
+        </div>  
+    </div>
 </main>
 
-<script src="../scripts/statsWithChartjs.js?<?= @filemtime(PUBLIC_PATH . '/scripts/statsWithChartjs.js') ?>"></script>  
\ No newline at end of file
+<script src="../scripts/statsWithChartjs.js?<?= @filemtime(PUBLIC_PATH . '/scripts/statsWithChartjs.js') ?>"></script>
```

Comment:
Please make sure to enable https://editorconfig.org in your IDE to use the correct whitespace conventions for this project.
For instance: new line at the end of the file, tabs a bit higher up.

Reply:
Thank you! I’ve enabled it, awesome tool!